docs/conf.py

```diff
@@ -83,7 +83,7 @@
 # -- Options for HTML output -------------------------------------------------
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ["_templates"]
-html_static_path = ["_static"]
```

Comment:
was there a warning related to this one?

Reply:
Yes.  I'm pretty sure "_static" is the default, but there isn't any _static folder relative to conf.py.  The exact error was:

`11:28:00 AM: copying static files... WARNING: html_static_path entry '/opt/build/repo/docs/_static' does not exist`